spacy/kb.pxd

```diff
@@ -28,8 +28,9 @@ cdef class Candidate:
 
 cdef class KnowledgeBase:
     cdef Pool mem
-    cpdef readonly Vocab vocab
+    cpdef public Vocab vocab
```

Comment:
```suggestion
    cdef readonly Vocab vocab
```

We'll normally want to make this readonly, as people could get segfaults if they mess with it.